src/olympia/amo/utils.py

```diff
@@ -1165,7 +1165,7 @@ def extract_colors_from_image(path):
 def use_fake_fxa():
     """Return whether or not to use a fake FxA server for authentication.
     Should always return False in production"""
-    return settings.DEBUG and settings.USE_FAKE_FXA_AUTH
+    return settings.DEV_MODE or settings.USE_FAKE_FXA_AUTH
```

Comment:
This wasn't connected to the statics problem I found, was it?  This change means you can't use real FxA auth in dev mode.

Reply:
No, I just needed that for local testing.. I will drop that before merging. THough it does kind of suck. Without that it mneans you HAVE to use real FxA in prod mode, which kind of also sucks. I think the real fix is to only rely on the USE_FAKE_FXA_AUTH and just make sure we hardcode production and base settings to prefer real auth. Or maybe rely on the SITE_URL or something.

Reply:
Probably out of scope for this PR, but we could rely on the client_id/secret being set to a real value - the default unless you've defined credentials in your `local_settings` or env is `.`.  Otherwise, redirecting to real FxA just ends in an error page.

Reply:
That's a great idea. Yeah should be a separate ticket but that makes a lot more sense, if you pass credentials, use em.